### ISSUE TITLE
Playlist populations aren't queried after matchmaking every 5 seconds anymore

### ISSUE THREAD

**Issue by mgavin** [mgavin/ShowPlayerPopulation#2]
Definitely happened with the last patch on 6/5/2024, Season 15 update.

Affects the latest release (at the time of writing): [3a569e5](https://github.com/mgavin/ShowPlayerPopulation/commit/3a569e57030ff7f0542b198f94f3f53a924402f6), dev commit: [bf0e09e](https://github.com/mgavin/ShowPlayerPopulation/commit/bf0e09ed70f9cc06f49093e794c2689ebb91e391)

Need to find a way around it.

Ugh. This sucks because this feels like I was targetted because I found this method... (and then showed it off on the discord, and then tried to submit it as a plugin.)

## IT'S ANNOYING BECAUSE:
  - THERE MAY BE A LOT MORE UNNECESSARY OUTPUT IN THE CONSOLE LOG WHICH **DOES NOT** NEED TO HAPPEN
    - because of the fucking matchmakingwrapper logging unnecessary things.
      I really hate this and would have to basically modify - in memory - where it's being called and replace it with null.
      And that would never get approved. Or I would have TO FUCKING EXPLAIN IT WHICH IS IMPOSSIBLE GIVEN THEIR SYSTEM.
  - This affects the graphing because it's not consistently automatic anymore. GDI.

## Maybe: 
  - Set a timer? Only be able to ask every 5 minutes? or... some set timer?
  - Find another function to hook?
    -  There are other functions related to populations... might be ... well, file will be included.
[functions.txt](https://github.com/user-attachments/files/15791783/functions.txt)
![image](https://github.com/mgavin/ShowPlayerPopulation/assets/1364492/2da84c35-38a9-409b-84a3-956d4a9be891)

## Also
  - The RL SDK might have more information about getting particular platform data, although I doubt it. It would be interesting to know what some of these functions returned.
... supposedly 16 hookable functions with "population" in the title

**Comment by mgavin** [comment]
I have no idea on an ETA. (which likely means within the week... 6/17/2024 ? ).

There were so many other things I wanted to move on to.

**Comment by mgavin** [comment]
Omfg.

This is because I tried to use [this queueing mechanism](https://github.com/mgavin/ShowPlayerPopulation/blob/3a569e57030ff7f0542b198f94f3f53a924402f6/ShowPlayerPopulation/ShowPlayerPopulation.cpp#L444).

If you use:

```
matchmakingwrapperobj.StartMatchmaking(/* BS */);
matchmakingwrapperobj.CancelMatchmaking();
```

the game will send you the playlist population on that five second interval again. Siiiiiiiigh. Will fix in next release.

**Comment by mgavin** [comment]
Fixed here: a383147
